### DISCUSSION TITLE
[Question][Hw6] Why my running time is less than 24 miniutes?

### DISCUSSION THREAD

**Discussion by gayeun4195** [prosyslab-classroom/cs348-information-security#436] [Q&A]
Name: Gaeun Noh

In the document, it says "Note that grading takes at least 24 minutes in total". However, my recent submissions take time about 15 minutes. (My score is not that high, though) What is the reason? Does it mean my code is interrupted due to some errors? 

Thank you for the answer in advance:)

**Comment by gayeun4195** [comment]
links of my weird submissions are below

- https://www.gradescope.com/courses/516803/assignments/2901334/submissions/182012879
- https://www.gradescope.com/courses/516803/assignments/2901334/submissions/182012434

**Comment by jiwoojeong17** [comment]
My submission took very few minutes when there were some errors. (e.g. argument error, length error, ...)
So checking one more time may solve the problem.
